mmeds/resources/summary_code.txt

```diff
@@ -35,13 +35,15 @@ for group_name in metadata_discrete:
     if group_name in mdf and not mdf[group_name].isnull().all():
         grouping = mdf[group_name]
         max_colors += grouping.nunique()
+        # Total number of discrete colors == number of discrete values across all vars
```

Comment:
this comment could be better, in general I think comments should:
1. come before code blocks
Note that additional comments can exist inside the code block, but should be on the shorter side and build on what was said in the initial comment for the code block
2. use plain english, I think pseudocode is harder to parse at a glance which is the main purpose of having comments.

Reply:
I realize there is a comment before the if/else block but it is lacking. 
# Create information for R color palettes

What information? what exactly is group_name supposed to be? 
With more context I can figure it out, like that mdf is the qiime_mapping_file (doesn't strike me as a great variable name)